src/API/Endpoints/Reports/RecentDonations.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Recent Donations endpoint
+ *
+ * @package Give
+ */
+
+namespace Give\API\Endpoints\Reports;
+
+class RecentDonations extends Endpoint {
+
+	public function __construct() {
+		$this->endpoint = 'recent-donations';
+	}
+
+	public function get_report( $request ) {
+
+		// Setup donation query args (get sanitized start/end date from request)
+		$args = [
+			'number'     => 50,
+			'paged'      => 1,
+			'orderby'    => 'publish_date',
```

Comment:
@henryholtgeerts What it typo in `orderby` value. I think this should be `post_date`. 
Let me know if I am missing something.

Reply:
Change made, ref: https://github.com/impress-org/givewp/pull/4429/commits/79b15b18133e870bef2bb73e30dd6ef744ea2eed